internal/db/migrations/2024-08-28.0-tss_transactions.sql

```diff
@@ -0,0 +1,37 @@
+-- +migrate Up
+
+CREATE TABLE tss_transactions (
+    transaction_hash VARCHAR(70) PRIMARY KEY,
+    transaction_xdr VARCHAR(400),
+    webhook_url VARCHAR(250),
+    current_status VARCHAR(50),
+    creation_time TIMESTAMPTZ DEFAULT NOW(),
+    last_updated_time TIMESTAMPTZ DEFAULT NOW(),
```

Comment:
Let's be consistent in all tables
```suggestion
    created_at TIMESTAMPTZ DEFAULT NOW(),
    updated_at TIMESTAMPTZ DEFAULT NOW(),
```